cabot/cabotapp/management/commands/create_superuser_on_heroku.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.auth.models import User
```

Comment:
This management command sounds useful with or without heroku. Can we rename it `create_superuser_from_environment_variables.py` ?